charts/opensrp-server-web/templates/config/opensrp/_opensrp.properties.tpl

```diff
@@ -53,6 +53,8 @@ couchdb.atomfeed-db.revision-limit={{ .Values.couchdb.atomfeed_db_revision_limit
 #RapidPro settings
 rapidpro.url={{ .Values.rapidpro.url }}
 rapidpro.token={{ .Values.rapidpro.token }}
+rapidpro.project={{ .Values.rapidpro.project }}
+rapidpro.listener.cron.schedule={{ .Values.rapidpro.listener_cron_schedule }}
```

Comment:
This property is named with underscore. Is it what is desired?

Reply:
@ellykits this will not affect the property as its a variable only within the chart